vm/src/builtins/str.rs

```diff
@@ -1692,7 +1692,7 @@ impl AnyStr for str {
         F: Fn(&Self) -> PyObjectRef,
     {
         // CPython split_whitespace
-        let mut splited = Vec::new();
+        let mut splitted = Vec::new();
```

Comment:
In this function, I'd recommend using `splits` instead of the nonstandard "splitted".